spec/models/item_spec.rb

```diff
@@ -0,0 +1,66 @@
+require 'rails_helper'
+
+describe Item, :type => :model do
+
+  it 'is invalid without a title' do
+    item = Item.new(title: nil)
+    expect(item).not_to be_valid
+  end
+
+  it 'is invalid without a description' do
+    item = Item.new(description: nil)
+    expect(item).not_to be_valid
+  end
+
+  it 'is invalid without a price' do
+    item = Item.new(unit_price: nil)
+    expect(item).not_to be_valid
+  end
+
+  it 'is valid with all three attributes' do
+    item = Item.new(title: "Something",
+                    description: "Thing",
+                    unit_price: 1)
+    expect(item).to be_valid
+  end
+
+  it 'cannot have a blank title' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.